.github/fixtures/test-latest-with-one-tag/commit.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-GIT_COMMITTER_DATE="2021-01-23 01:23:45" git commit --allow-empty -m "Initial commit"
+GIT_COMMITTER_DATE="2021-01-23 01:23:45" git commit --allow-empty -m "feat: initial commit"
```

Comment:
I just tried out this fixture without changing the rest of the code and I'm getting the same result. Can you check if the behavior change is correct here?

Reply:
Oops nvm, I was not using `--latest` :)